types/matomo-tracker-react-native/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for matomo-tracker-react-native 0.3
+// Project: https://github.com/donni106/matomo-tracker-react-native
+// Definitions by: Zeynep Ece Ergin <https://github.com/zecergin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import * as React from 'react';
+
+export interface AppStart {
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Action {
+  name: string;
+  userInfo?:
+  {
+    uid?: string
+  };
+}
+
+export interface Event {
+  category: string;
+  action: string;
+  name?: string;
+  value?: number;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface SiteSearch {
+  keyword: string;
+  category?: string;
+  count?: number;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Link {
+  link: string;
+  url?: string;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Download {
+  download: string;
+  url?: string;
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export function useMatomo(): {
+  trackAppStart: (params: AppStart) => undefined | Promise<Response>,
+  trackScreenView: (params: Action) => undefined | Promise<Response>,
+  trackAction: (params: Action) => undefined | Promise<Response>,
+  trackEvent: (params: Event) => undefined | Promise<Response>,
+  trackSiteSearch: (params: SiteSearch) => undefined | Promise<Response>,
+  trackLink: (params: Link) => undefined | Promise<Response>,
+  trackDownload: (params: Download) => undefined | Promise<Response>
+};
+
+export interface MatomoProviderProps {
+  instance: MatomoTracker;
+  children: React.ReactElement;
+}
+
+export function MatomoProvider(props: MatomoProviderProps): JSX.Element;
+
+export const MatomoContext: React.Context<{}>;
+export interface InstanceProps {
```

Comment:
```suggestion

export interface InstanceProps {
```
space, be consistent